Assets/RGLUnityPlugin/Scripts/LowLevelWrappers/RGLNativeAPI.cs

```diff
@@ -510,9 +517,24 @@ public static void NodePointsFilterGround(ref IntPtr node, float groundAngleThre
             }
         }
 
-        public static void NodePointsRadarPostprocess(ref IntPtr node, float distanceSeparation, float azimuthSeparation)
+        public static void NodePointsRadarPostprocess(ref IntPtr node, RadarParametersScope[] radarParametersScopes,
+            float rayAzimuthStep, float rayElevationStep, float frequency)
         {
-            CheckErr(rgl_node_points_radar_postprocess(ref node, distanceSeparation, azimuthSeparation));
+            RadarParametersScope[] radarParametersScopesCopy = new RadarParametersScope[radarParametersScopes.Length];
+            radarParametersScopes.CopyTo(radarParametersScopesCopy, 0);
```

Comment:
You may remove this copy at all and assign values below. Nitpick obviously, but will be a little bit more clear.